tensorbay/opendataset/nuImages/loader.py

```diff
@@ -110,7 +110,12 @@ def _get_annotation_info(info_path: str, is_test: bool = False) -> Dict[str, Any
             info_path, "surface_ann", no_key_frame=True, determined_token="sample_data_token"
         )
         annotation_info["category"] = get_info_with_token(info_path, "category")
-        annotation_info["attribute"] = get_info_with_token(info_path, "attribute")
+        attribute_details = get_info_with_token(info_path, "attribute")
+        for attribute_token in attribute_details.keys():
+            attribute_details[attribute_token]["name"] = attribute_details[attribute_token][
+                "name"
+            ].replace(".", "-")
+        annotation_info["attribute"] = attribute_details
```

Comment:
Should we split the original attribute name by `.` and store the name with value in the attribute_details?
```
key, value = attribute_details[attribute_token]["name"].rsplit(".", 1)
attribute_details[attribute_token]["name_and_value"] = key.replace(".", "-"), value
```